packages/SwingSet/misc-tools/prune-transcripts.js

```diff
@@ -0,0 +1,74 @@
+// @ts-nocheck
+import '@endo/init';
+import { performance } from 'perf_hooks';
+import fs from 'fs';
+import sqlite3 from 'better-sqlite3';
+
+// Given a swingstore.sqlite filename, prune all the old (historical,
+// not-current-span) transcript entries from it. Note that this is destructive.
+//
+// With --vacuum-into FN, it will then produce a new DB that has all
+// the same (pruned) data, but is a lot smaller.
+
+let [dbfn, ...args] = process.argv.slice(2);
+let intofn;
+if (args.length >= 2) {
+  if (args[0] === '--vacuum-into') {
+    intofn = args[1];
+    args = args.slice(2);
+  }
+}
```

Comment:
```suggestion
// With `--vacuum-into dest.sqlite`, it will then produce a new DB that has all
// the same (pruned) data, but is a lot smaller.

const { _: args, ...options } = yargsParser(process.argv.slice(2));
let dbfn = args.shift();
const intofn = options['vacuum-into'];
```